dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -16,14 +16,14 @@
 : "${RIOTBASE:="$(cd "$(dirname "$0")/../../../" || exit; pwd)"}"
 
 : "${RIOTTOOLS:=${RIOTBASE}/dist/tools}"
+# shellcheck source=../ci/github_annotate.sh
```

Comment:
What's the purpose of the comment?

Reply:
To [fix SC1091](https://www.shellcheck.net/wiki/SC1091)